wiki/Installation/Linux/en.md

```diff
@@ -0,0 +1,351 @@
+# Linux
+
+*Main page: [Installation](/wiki/Installation).*
+
+This page will roughly tell you how to get osu! working on your Linux device using Wine.
+
+## Prerequisite
+
+Before installing osu!, you will need to install Wine first. Find your Linux distribution below for instructions on how to do this. If your Linux distribution is not listed, please look up on how to do this then update this article with the commands and packages needed to install to get osu! working.
+
+### Arch Linux
+
+1.  If you are using 64-bit Arch, enable Multilib. See [Multilib (ArchWiki)](https://wiki.archlinux.org/index.php/multilib) for more information.
+
+2.  Open the terminal and enter as root. Then install `wine`, `winetricks`, `lib32-gnutls`, and `wget`. `wget` is optional, but this guide will be using it.
+```
+$ su
+# pacman -S wine winetricks lib32-gnutls wget
+# exit
+```
+
+3.  Continue to [Installing osu!](#installing-osu!)
```

Comment:
To be sure, there shouldn't be a period when osu! ends?
Will do this on the formatting PR to remove period instances.